### ISSUE TITLE
Using Pytorch's Learning Rate Scheduler. How are detectron2's iters related to epochs?

### ISSUE THREAD

**Issue by oakhtar147** [facebookresearch/detectron2#2058]
I wanted to implement Pytorch's `MultiStepLR(optimizer, milestones=[30,80], gamma=0.1, last_epoch=-1)` but it is confusing since the model in detectron2 is based on iters and not epochs. How do I use the scheduler with my `DefaultTrainer` object? What arguments do I pass to the  `MultiStepLR(optimizer, milestones=[30,80], gamma=0.1, last_epoch=-1)` `milestones` and `last_epoch` parameters?

**Comment by ppwwyyxx** [comment]
With `milestones=[30,80]`, learning rate will be changed at the 30th and 80th iteration.
`last_epoch=k` means starting from the `kth` iteration.
